# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.h
# Commit: ae4f46dfec58
# Full Hash: ae4f46dfec5886a64797be7073f63bda296c3541
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-29 20:54 +0000
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P1: Implement details relationship for CSS anchored local accessibles. r=Jamie,emilio
# ==============================================================================

diff -r 55449173b0e6 -r ae4f46dfec58 accessible/generic/LocalAccessible.h
--- a/accessible/generic/LocalAccessible.h	Mon Sep 29 20:39:23 2025 +0000
+++ b/accessible/generic/LocalAccessible.h	Mon Sep 29 20:54:29 2025 +0000
@@ -1048,6 +1048,8 @@
   LocalAccessible* GetCommandForDetailsRelation() const;
 
   LocalAccessible* GetPopoverTargetDetailsRelation() const;
+
+  LocalAccessible* GetAnchorPositionTargetDetailsRelation() const;
 };
 
 ////////////////////////////////////////////////////////////////////////////////